### DISCUSSION TITLE
Guidance on 3.30.17 and 3.30.16 beta releases for 2021 West Coast Groundfish assessments?

### DISCUSSION THREAD

**Discussion by iantaylor-NOAA** [nmfs-stock-synthesis/stock-synthesis#428] [Team Posts]
The assessment folks at NWFSC are busy with a collection of assessments, with the first pre-review reports due in mid-April and the last ones due in early July.

I got a question about an incorrect count of warnings being reported, which was an issue that seems to have been fixed in 3.30.16.04, but only 3.30.16.00 and 3.30.16.02 are available on VLab. The warnings issue is easy to ignore but it got me wondering about the other changes that have occurred over the past few months.

Would any of the following make sense:

1. suggest that folks could download newer versions of 3.30.16 as compiled by GitHub Actions either as a replacement for the version they are using or to confirm that there are no significant differences from what they are currently using?
2. upload newer bug-fix versions of 3.30.16 to VLab?
3. not suggest any version change until the 3.30.17 release, at which point whatever assessments are still in process could switch over?

**Comment by Rick-Methot-NOAA** [comment]
I recommend #3 so they can have a better reference for which model version they are using.  Let's set a drop dead date for changes to include in .17.
I think I just want to merge Maunder's M branch into main, then do a look through open issues for anything pressing to get finished.

**Comment by k-doering-NOAA** [comment]
Totally agree. 

I think it is fine if folks want to check out a recent github actions compiled exe so they know what to expect with .17, but it definitely shouldn't be used in an assessment since it isn't an official version.

**Comment by iantaylor-NOAA** [comment]
OK, sounds good. I don't think any of the changes since 3.30.16.00 will significantly impact these west coast models.
